languages/en.ini

```diff
@@ -1415,6 +1415,8 @@ total_resources = "Total Resources"
 total_saved_items = "Total Saved Items"
 total_tags = "Total Tags"
 total_users = "Total Users"
+transit_sent_on = "sent on"
```

Comment:
For internationalization, it would be better to use strings with tokens instead of concatenating things in the templates... so, for example, `transit_to` could be changed from `to` to `to %%location%%` and then when you translate it in the template, you could do something like:

`<?=$this->transEsc('transit_to', ['%%location%%' => $item['in_transit'])?>`

Reply:
Even the `to %%location%%` is problematic in some languages, being a partial sentence. It also assumes that the status is compatible with it, which isn't necessarily the case. I'd suggest something like
`transit_to = "Destination: %%location%%"`
or hiding status and making it
`transit_to = "In transit to %%location%%"`